style.css

```diff
@@ -0,0 +1,186 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 16px;
+    background-color: #FE9D8D;
+    font-family: "Press Start 2P", sans-serif;
+}
+
+h1 {
+    color: #373C99;
+}
+
+.landingTopContainer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.landingLeft {
+    width: 20%;
+}
+
+.landingTitleContainer {
+    justify-content: space-between;
+}
+
+.leaderSurfboardContainer {
+    display: flex;
+}
+
+.mainContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+#mainGameContainer {
+    margin: 5%;
+    width: 1000px;
+    height: 600px;
+    position: relative;
+    border-radius: 10px;
+    background: linear-gradient(#f9855d,#fdd288,#ffffff);
+    overflow: hidden;
+}
+
+.waves {
+    position: relative;
+    width: 100%;
+    height: 30vh;
+    margin-bottom:-7px;
+    min-height:100px;
+    max-height:150px;
+    z-index: -2;
+    
+}
+
+.sun {
+    width: 100px;
+    height: 100px;
+    position: absolute;
+    top: 370px;
+    right: 40px;
+    background-color: #f9cb5d;
+    border-radius: 50%;
+    z-index: 2;
+}
+
+.ocean {
+    height: 150px; /* change the height of the waves here */
+    width: 400%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    overflow-x: hidden;
+    background-color: #1e90ff;
+    border-top: 2px #ffffff solid;
+    z-index: 4;
+}
+
+#avatar {
+    width: 60px;
+    height: 40px;
+    position: absolute;
+    left: 0px;
+    top: 415px;
+    z-index: 5;
+    background-image: url(./imgs/surfingPenguin.png);
+    background-size: cover;
+}
+
+#displaySplashInstructionsContainer {
+    position: absolute;
+    display: flex;
+    top: 0px;
+    left: 0px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 600px;
+    width: 1000px; 
+    padding: 40px;
+    background-color: #FE9D8D;
+    border: 5px #373C99 solid;
+    z-index: 10;
+    /* display: none; */
+    /*comment this out as needed */
+}
+
+#leaderSurfboard {
+    border-radius: 10px;
+    background-color: #373C99;
+    color: #F0EFF5;
+    width: 150px;
+    margin-right: 50px;
+}
+
+.startGameButtonContainer {
+    display: flex;
+    justify-content: center;
+}
```

Comment:
Does this need to exist? Not sure why the button has a container or why it's using flex

Seems to me like the flex rules you already have on the instructions container will do what you want without this extra div

Reply:
Removed from HTML and CSS.